Engineering-Reference/M365SourceAO.md

```diff
@@ -357,31 +336,31 @@ Default Value: On
 
 Migrate user account delegation.
 
-Change Conditions: ??
+Change Conditions: Turn off to disable migration of account delegation.
 
 ### Migrate Out Of Office <a name="migooo"></a>
 {: .no_toc }
 Default Value: On
 
 Migrate user Out Of Office settings.
 
-Change Conditions: ??
+Change Conditions: Turn off to disable migration of account OoO settings.
```

Comment:
Don't think this even is possible .   But should read "Turn off to disable migration of account Out of Office settings

Reply:
it is possible, tested it the other day. Made the change.